databases/functions/userBlockUser/user-block.js

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The ESLint disable comment is unnecessary here. It seems like the `Model` from 'sequelize' is not being used anywhere in this file. If that's the case, you should remove the import instead of disabling the linter warning.
```diff
- // eslint-disable-next-line no-unused-vars
- const {Model} = require('sequelize');
+ // Removed unused import
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->